### ISSUE TITLE
"Start a group" not redirecting correctly

### ISSUE THREAD

**Issue by taoeffect** [okTurtles/group-income#161]
### Problem

With latest changes in #155 (commit eb64870).

Multiple issues with "Start a group":

1. ~~Clicking "Start a group" highlights "New User" menu item instead of "Start a group"~~ **EDIT:** Err, I guess this makes sense since we're on the SignUp view.
2. Proceeding with "Sign Up" results in error. Tried this twice, screenshots from both attempts:

![screen shot 2017-01-05 at 7 50 31 pm](https://cloud.githubusercontent.com/assets/138706/21703513/9db88a4c-d380-11e6-9b4d-58d09ca1a46e.jpg)

![screen shot 2017-01-05 at 7 53 58 pm](https://cloud.githubusercontent.com/assets/138706/21703527/ca279942-d380-11e6-8bf8-fd279b25de87.jpg)

Meanwhile, the server console outputs:

```
Running "watch" task
Waiting...
Gruntfile.js: loading file with babel support: ./backend/index.js
API server running at: http://Alice.local:3000
Debug: handler, error
    {"msec":12.488315999507904,"error":"Not Found","data":{"data":null,"isBoom":true,"isServer":false,"output":{"statusCode":404,"payload":{"statusCode":404,"error":"Not Found"},"headers":{}},"message":"Not Found"}}
```

### Solution

1. ~~Clicking "Start a group" should highlight "Start a group" tab/menu~~
2. Clicking "Sign Up" should redirect to group creation screen. Right now it's trying to load `localhost:8000/user/` for some reason, and it also fails to load even that.

**Comment by taoeffect** [comment]
**Added label `Kind:Test`:** Should probably create a test for this as well so that we can automatically check this for correct behavior in the future.

**Comment by vijayee** [comment]
It is not redirecting to the `localhost:3000/user/` but failing to make the xhr request to the backend at that route due to it being removed. It now fails the CORS request. It should redirect correctly once a new login scheme is created.

**Comment by vijayee** [comment]
Fixed in #186
